storybook/stories/LabeledItem/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import LabeledItem from '../../../src/components/LabeledItem'
+
+export default function StoryLabeledItem ({ children, labelAlign, labelTextAlign, labelType, labelText }) {
+  return (
+    <LabeledItem labelAlign={labelAlign} labelTextAlign={labelTextAlign} labelType={labelType} labelText={labelText}>
+      {children}
+    </LabeledItem>
+  )
+}
+
+StoryLabeledItem.defaultProps = {
+  labelAlign: 'left',
+  labelTextAlign: 'left',
+  labelType: 'currencyColored',
+  labelText: 'Test Label',
```

Comment:
It must be empty string in production. For StoryBook this props will be set on a story's screen.

Reply:
there are default props, when user missed to input something